services/py-api/.env.sample

```diff
@@ -1,4 +1,5 @@
 ENV=DEV
 PORT=8080
-ADDRESS=localhost
+# means "all IPv4 addresses on the local machine". This is needed for Docker.
+ADDRESS="0.0.0.0"
```

Comment:
Why is this needed for docker?

Reply:
If I run the image with `-p 127.0.0.1:8080:8080`, I am not able to access the API when pinging it on `https://localhost/api/v3/ping`. It says connection was closed.

Reply:
Okay. I read a bit more about it. That's fine security wise.

Reply:
Btw, I'm seeing you have `v3` in your urls. Does it make sense to continue the versioning if both the other two APIs are deprecated and discarded?

Reply:
> Btw, I'm seeing you have `v3` in your urls. Does it make sense to continue the versioning if both the other two APIs are deprecated and discarded?

I left it on purpose, as a jem :D. To remind us that we had 2 previous APIs.

Reply:
> Why is this needed for docker?

I confirm this because when I was setting the dev container I had to change the address to 0.0.0.0 for the localhost to work.